dvc/scm/base.py

```diff
@@ -24,6 +24,16 @@ class RevError(SCMError):
     pass
 
 
+class NOSCMError(SCMError):
+    def __init__(self):
+        msg = (
+            "only supported for Git repositories. If you had\n already "
+            "initialized a git repo, this may be caused by dvc configuration. "
+            "Use\n `dvc config core.no_scm False` to update it."
```

Comment:
Shouldn't this begin with capital O? Depends if it's printed by itself (in that case yes) or if it's a continuation of a previously printed message (in that case no). Other capitalizations:
```suggestion
            "only supported for Git repositories. If you had\n already "
            "initialized a Git repo, this may be caused by DVC configuration. "
            "Use\n `dvc config core.no_scm False` to update it."
```

Reply:
> If you had already initialized a Git repo, this may be caused by DVC configuration...

I'm not sure I understand this explanation though. I think we mean something like ``If you're seeing this error in a Git repo, try updating the DVC configuration with `dvc config core.no_scm False`.`` ?

Reply:
I think so, my pool english representation.

> > If you had already initialized a Git repo, this may be caused by DVC configuration...
> 
> I'm not sure I understand this explanation though. I think we mean something like `` If you're seeing this error in a Git repo, try updating the DVC configuration with `dvc config core.no_scm False`. `` ?

Reply:
Ah sorry I didn't realize you were an outside contributor! Don't worry about it we can polish it 🙂 this is a great change. Thanks a lot!

Reply:
Thank you too.